src/grafana/prometheus/Cadence-Server-Basic.json

```diff
@@ -1396,21 +1458,3908 @@
+        "show": true,
+        "values": []
+      },
+      "yaxes": [
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        },
+        {
+          "format": "short",
+          "label": null,
+          "logBase": 1,
+          "max": null,
+          "min": null,
+          "show": true
+        }
+      ],
+      "yaxis": {
+        "align": false,
+        "alignLevel": null
+      }
+    },
+    {
+      "aliasColors": {},
+      "bars": false,
+      "dashLength": 10,
+      "dashes": false,
+      "datasource": "Prometheus",
+      "fieldConfig": {
+        "defaults": {
+          "links": []
+        },
+        "overrides": []
+      },
+      "fill": 1,
+      "fillGradient": 0,
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 12,
+        "y": 109
+      },
+      "hiddenSeries": false,
+      "id": 90,
+      "legend": {
+        "avg": true,
+        "current": true,
+        "max": true,
+        "min": true,
+        "show": true,
+        "total": false,
+        "values": true
+      },
+      "lines": true,
+      "linewidth": 1,
+      "nullPointMode": "null",
+      "options": {
+        "alertThreshold": true
+      },
+      "percentage": false,
+      "pluginVersion": "8.2.1",
+      "pointradius": 2,
+      "points": false,
+      "renderer": "flot",
+      "seriesOverrides": [],
+      "spaceLength": 10,
+      "stack": false,
+      "steppedLine": false,
+      "targets": [
+        {
+          "expr": "histogram_quantile($quantiles, (sum by (le, operation) (persistence_latency_bucket{namespace=\"$namespace\", cadence_service=\"cadence_frontend\"}))) ",
+          "hide": false,
+          "interval": "",
+          "legendFormat": "cadence_frontend-{{operation}}",
+          "refId": "A"
+        },
+        {
+          "expr": "histogram_quantile($quantiles, (sum by (le, operation) (persistence_latency_bucket{namespace=\"$namespace\", cadence_service=\"cadence_matching\"}))) ",
+          "interval": "",
+          "legendFormat": "cadence_matching-{{operation}}",
+          "refId": "B"
+        },
+        {
+          "expr": "histogram_quantile($quantiles, (sum by (le, operation) (persistence_latency_bucket{namespace=\"$namespace\", cadence_service=\"cadence_history\"}))) ",
+          "interval": "",
+          "legendFormat": "cadence_history-{{operation}}",
+          "refId": "C"
+        },
+        {
+          "expr": "histogram_quantile($quantiles, (sum by (le, operation) (persistence_latency_bucket{namespace=\"$namespace\", cadence_service=\"cadence_worker\"}))) ",
+          "interval": "",
+          "legendFormat": "cadence_worker-{{operation}}",
+          "refId": "D"
+        }
+      ],
+      "thresholds": [],
+      "timeFrom": null,
+      "timeRegions": [],
+      "timeShift": null,
+      "title": "Persistence By Operation Latency(in seconds)",
```

Comment:
is this also by seconds?

Reply:
I'm not completely sure since the exported metrics have quite different values. I attach a copy of the data
<img width="1637" alt="Captura de pantalla 2021-12-01 a las 0 53 35" src="https://user-images.githubusercontent.com/47592240/144147150-e30d324c-d8a5-48d2-a62a-a664a34f586b.png">
.